.github/workflows/automated-create-release-branch.yml

```diff
@@ -0,0 +1,19 @@
+name: Automated Create Release Branch
+
+on:
+  # Automated trigger from a job running on an external repo
+  repository_dispatch:
+    types:
+      - automated-create-release-branch
```

Comment:
Can't you just add this in addition to `workflow_dispatch` in the existing script?

Reply:
I don't think so, the way to access the inputs is different

Reply:
Hmm yeah I see that now. Is is possible to use something like`${{ inputs.adapter-version || github.event.client_payload.adapter-version }}` instead?

Reply:
oh duh, that could work. let me try that in the reference adapter

Reply:
Can you take a look at https://github.com/ChartBoost/chartboost-mediation-ios-adapter-reference/pull/63?

Reply:
It works! I'll apply to all adapters